hack/patch-precache-image.py

```diff
@@ -11,4 +11,4 @@
         item['image'] = pull
     break
 with open(file,'w') as f:
-    yaml.dump(obj, f)
+    yaml.dump(obj, f, default_flow_style=False)
```

Comment:
I assume you set default_flow_style to false so written yaml files would also have the block style.

Reply:
Right, so [config/manifests/pre-cache-image-patch.yaml](https://github.com/openshift-kni/cluster-group-upgrades-operator/pull/83/files#diff-4c8b8ab4c6fed062d178c60b264310405d5163ec652a8535ca9ea2fde1affe12) is the written file. It is created when you run make bundle.
Thanks!

Reply:
Awesome, thanks! ;)